index.html

```diff
@@ -145,21 +145,21 @@ <h2 class="about-me-headline">About<br>Myself</h2>
             don’t hestiate to contact me.</p>
             <p class="social2">LET'S CONNECT</p>
             <ul class="social-icons2">
-                <li class="social-icons-items"><a href=""><img src="images/twitter.png" alt=""></a></li>
-                <li class="social-icons-items"><a href=""><img src="images/linkedin.png" alt=""></a></li>
-                <li class="social-icons-items"><a href=""><img src="images/M.png" alt=""></a></li>
-                <li class="social-icons-items"><a href=""><img src="images/github.png" alt=""></a></li>
-                <li class="social-icons-items"><a href=""><img src="images/hand.png" alt=""></a></li>
+                <li class="social-icons-items"><a href=""><img src="images/twitter.png" alt="twitter profile link"></a></li>
+                <li class="social-icons-items"><a href=""><img src="images/linkedin.png" alt="LinkedIn profile link"></a></li>
+                <li class="social-icons-items"><a href=""><img src="images/M.png" alt="M profile link"></a></li>
+                <li class="social-icons-items"><a href=""><img src="images/github.png" alt="GitHub profile link"></a></li>
+                <li class="social-icons-items"><a href=""><img src="images/hand.png" alt="Hand profile link"></a></li>
             </ul>
             <button class="resume-btn">Get my resume</button>
         </div>
 
         <div class="aboutme-desk-right">
             <p class="languages">Languages <a href=""><img src="images/down-arrow.png" alt=""></a></p>
             <ul class="languages-list">
-                <li class="languages-list-item"><img src="images/js.png" alt="">JavaScript</li>
-                <li class="languages-list-item"><img src="images/html5.png" alt="">HTML</li>
-                <li class="languages-list-item"><img src="images/css3.png" alt="">CSS</li>
+                <li class="languages-list-item"><img src="images/js.png" alt="javascript logo">JavaScript</li>
+                <li class="languages-list-item"><img src="images/html5.png" alt="html5 logo">HTML</li>
+                <li class="languages-list-item"><img src="images/css3.png" alt="css3 logo">CSS</li>
             </ul>
             <p class="under-languages">Frameworks <a href=""><img src="images/right-arrow.png" alt=""></a></p>
```

Comment:
- [ ] As I can see here you didn't give alternative text to your image but you should give all your images descriptive alt text for accessibility reason so kindly checkk this lines :  `49 `, `51` , `73`, `75`,  `97` , `99` , `121` . `123` , `158`, `164` , `166`. so kindly add an alt text to all this lines that contain image because I see them empty now .😊💯🔥


![image](https://user-images.githubusercontent.com/58553711/200580623-c5927c7f-4c17-4e06-bf29-94aba4127037.png)

Reply:
@Reem-lab I understand the changes required above, but for this, it's only design images, that's just a separative dot you can't even describe it in the alt text